frontend/src/components/AddIngredients.js

```diff
@@ -3,20 +3,32 @@ import { TextField } from "@material-ui/core";
 import { Autocomplete } from "@material-ui/lab";
 import { useState } from "react";
 
-export default function AddIngredients({ ingredients, setIngredientsList }) {
+export default function AddIngredients({
+  ingredients,
+  ingredientsList,
+  setIngredientsList,
+}) {
   const [ingredient, setIngredient] = useState("");
+  const [description, setDescription] = useState("");
+  let checkForDouble = false;
 
   const handleAddClick = (event) => {
     event.preventDefault();
-    setIngredientsList((ingredientsList) => [
-      ...ingredientsList,
-      ingredient.name,
-    ]);
+    for (let i = 0; i < ingredientsList.length; i++) {
+      if (ingredientsList[i] === ingredient.name) {
+        checkForDouble = true;
+      }
+    }
+    if (checkForDouble === false) {
```

Comment:
```suggestion
    if (!checkForDouble) {
```
This is already boolean, no need to compare it to another boolean. Is probably not needed anyway, see comment above.